doc/source/release/1.16.0-notes.rst

```diff
@@ -58,11 +61,13 @@ New features
   deprecated and will be removed four versions later. Similarly, after
   changing the underlying algorithm to recursion, the ``blocksize`` keyword
   argument has no effect and will be removed two versions later.
-
+- Added wrappers for ``?langb`` and ``?gbcon`` to `scipy.linalg.lapack`.
```

Comment:
@j-boway I'm not sure if this is all of them. Thought you might want to take a look at the release notes.